airbyte-webapp/src/packages/cloud/views/auth/Auth.module.scss

```diff
@@ -26,10 +27,12 @@
   overflow: hidden;
 }
 
-@media (min-width: 992px) and (min-height: 720px) {
-  .container {
-    flex-direction: row;
-    height: 100%;
+@media (min-width: 992px) {
```

Comment:
:information_source: I removed the height requirement, so make sure we're only using width, since I believe we still want to build responsive breakpoints on those width purely.

Reply:
Do we want to set some breakpoints in `_variables`?

Reply:
Yes I do think we want so, but not as part of this PR. We should properly define and work with Nico on our responsive breakpoints, so this should be part of a separate PR with component. https://github.com/airbytehq/airbyte/issues/16357